homeworks/ekaterina.sharova_rogerosha/PokemonProject/style.css

```diff
@@ -1,22 +1,127 @@
+.upper-header {
+  background-color: #333;
+  display: grid;
+  align-items: center;
+  grid-template-columns: 1fr auto;
+  padding: 10px;
+  color: #333;
+  gap: 50px;
+  height: 80px;
+  justify-content: space-between;
+}
+
+.ma-logo {
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+}
+
+.MAlogo {
+  width: 29.11px;
+  height: 29.95px;
+  margin-left: 53px;
+}
+
+.MA {
+  width: 147.68px;
+  height: 9.08px;
+  margin-left: 13.21px;
+  margin-top: 10px;
+}
+
+.navigation {
+  text-align: right;
+  display: flex;
+  align-items: center;
+}
+
+.navigation ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.navigation-item {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  color: #FFF;
+  line-height: 19px;
+  margin-right: 50px;
+  display: flex;
```

Comment:
`display: flex;` - is a flex container for some blocks. If inside of the link is only text - no need to make it a flex

Reply:
can't delete it - everything going to be a mess then